packages/mcp-provider-dx-core/test/e2e/create_scratch_org.test.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2025, Salesforce, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { McpTestClient, DxMcpTransport } from '@salesforce/mcp-test-client';
+import { execCmd, TestSession } from '@salesforce/cli-plugins-testkit';
+import { z } from 'zod';
+import { createScratchOrgParams } from '../../src/tools/create_scratch_org.js';
+
+describe('create_scratch_org', () => {
+  const client = new McpTestClient({
+    timeout: 120_000,
+  });
+  
+  const devHubUsername = process.env.TESTKIT_HUB_USERNAME as string;
+
+  let testSession: TestSession;
+  let resolvedDevHubUsername: string;
+
+  const createScratchOrgSchema = {
+    name: z.literal('create_scratch_org'),
+    params: createScratchOrgParams,
+  };
+
+  before(async () => {
+    testSession = await TestSession.create({
+      project: {
+        name: 'MyTestProject',
+      },
+      devhubAuthStrategy: 'AUTO',
+    });
+
+    // Get the hub username from the testSession and set it as the global target-dev-hub
+    const hubUsername = testSession.hubOrg?.username;
+    resolvedDevHubUsername = hubUsername ?? devHubUsername;
+    
+    if (!resolvedDevHubUsername) {
+      throw new Error('No DevHub username available from TestSession or TESTKIT_HUB_USERNAME environment variable');
+    }
+
+    execCmd(`config set target-dev-hub ${resolvedDevHubUsername} --global`, {
+      cli: 'sf',
+      ensureExitCode: 0,
+    });
```

Comment:
testkit already set the hub as the global one.

can we simplify this by only accessing the hub username via `testSession.hubOrg.username`?
so, no `TESTKIT_HUB_USERNAME` lookup or `config get`